suite-common/token-definitions/src/__tests__/utils.test.ts

```diff
@@ -39,3 +41,25 @@ describe('getSupportedDefinitionTypes', () => {
         });
     });
 });
+
+describe('buildTokenDefinitionsFromStorage', () => {
+    const tokenDefinitions = buildTokenDefinitionsFromStorage(storageTokenManagement);
+    expect(tokenDefinitions).toEqual({
+        eth: {
+            coin: {
+                error: false,
+                data: undefined,
+                isLoading: false,
+                hidden: ['0xA', '0xB'],
+                shown: ['0xC', '0xD'],
+            },
+            nft: {
+                error: false,
+                data: undefined,
+                isLoading: false,
+                hidden: ['0xE', '0xF'],
+                shown: ['0xG', '0xH'],
+            },
+        },
+    });
```

Comment:
Please follow the style of the previous tests (result in fixtures)

Reply:
3ee8d004a81182df33771498b3093f505f150faf